spec/features/admin_views_categories_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature 'admin views list of categories', %Q{
+  As an admin, I want to view a list of all categories and subcategories, so
+  that I can see what type of content the site has.
+  Acceptance Criteria:
+  * [X] - I see a paginated list of categories ordered by name.
+  * [X] - I can see a list of subcategories belonging to a category.
+  } do
+
+  let(:admin) do
+    FactoryGirl.create(:user, admin: true)
+  end
+
+  before(:each) do
+    3.times do
+      FactoryGirl.create(:category)
+      3.times do
+        FactoryGirl.create(:category, parent: Category.cat_parents.last)
+      end
+    end
+  end
+
+  scenario 'admin views list of categories' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.